src/_beginmaxheaders.h

```diff
@@ -25,3 +25,9 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 //
+
+// Build options header.
+#include "foundation/core/buildoptions.h"
```

Comment:
I like these two headers included here but this header still needs to be included separately in many places.

Reply:
As far as I can tell it's included everywhere where it must be, can you point a place where that isn't the case?

Reply:
No @dictoon, it's all fine. It's just that _begin header little later includes it again. But that's not big deal. It was just a thought, sorry.

Reply:
No problem, thanks for the review!